src/commands/flex/plugins/start.ts

```diff
@@ -1,16 +1,69 @@
+import { flags } from '@oclif/command';
+
 import { createDescription } from '../../../utils/general';
-import FlexPlugin from '../../../sub-commands/flex-plugin';
+import FlexPlugin, { ConfigData, SecureStorage } from '../../../sub-commands/flex-plugin';
+
 /**
  * Starts the dev-server for building and iterating on a flex-plugin
  */
 export default class FlexPluginsStart extends FlexPlugin {
-  static description = createDescription('Starts a dev-server to build the Flex plugin locally', true);
+  static description = createDescription('Starts a dev-server to build the Flex plugin locally', false);
+
+  static flags = {
+    ...FlexPlugin.flags,
+    name: flags.string({
+      multiple: true,
+      char: 'n',
+    }),
+    'include-remote': flags.boolean(),
+  };
+
+  constructor(argv: string[], config: ConfigData, secureStorage: SecureStorage) {
+    super(argv, config, secureStorage, {});
+
+    if (this._flags['include-remote'] || this._flags.name) {
+      this.opts.runInDirectory = false;
+    }
+  }
 
   /**
    * @override
    */
   async doRun() {
-    await this.runScript('check-start');
-    await this.runScript('start');
+    const flexArgs: string[] = [];
+    const pluginArgs: string[] = [];
+
+    if (this._flags.name) {
+      for (const name of this._flags.name) {
+        flexArgs.push('--name', name);
+        if (!name.includes('@remote')) {
```

Comment:
Take a look at this page https://oclif.io/docs/flags - we can create custom flags and that might be a better way of doing this (i.e. `name` is a custom flag that would be parsed into an object that contains `name: string, version: string, remote: boolean`)

Reply:
I now think that we should not do this because we want the whole string of whatever the user typed sent to plugin-builder and the custom flag defeats the purpose of that